v1/ao/internal/reporter/context.go

```diff
@@ -603,6 +618,23 @@ func NewContext(layer string, reportEntry bool, opts ContextOptions,
 				SetHeaders(ttNotRequested)
 			}
 
+			return ctx, true, headers
+		}
+	} else if opts.Overrides.ExplicitMdStr != "" {
+		var err error
+		//take note that the ctx here will be the same as the entry event
+		//we might consider randomizing the opID or the set it to zero in the future
+		//for now we will just disable the opID check in reporter.prepareEvent as the Op ID will be the same
+		//for context and entry event
+		if ctx, err = NewContextFromMetadataString(opts.Overrides.ExplicitMdStr); err != nil {
+			log.Info("passed in x-trace seems invalid, ignoring")
```

Comment:
very minor: these three logs are not very informative (e.g., if an x-race `seems invalid`, it may be better to print it out as well)